_docs/automation-time-apis.md

```diff
@@ -121,6 +121,180 @@ pub enum Error {
 }
 ```
 
+### RPC: Generate Task ID based on Provided ID and Account ID
+This API allows you to generate the Task ID based on the Provided ID and Account ID used to create the task.
+
+#### Call
+```rust
+fn generate_task_id(
+    /// The `account_id` of the user. This is the wallet address of the user who created the task.
+    account_id: AccountId,
+    /// An id provided by the user. This id must be unique for each task for a given user.
+    provided_id: Vec<u8>,
+    )
+```
+
+** For those who are not familiar with rust `Vec<u8>` accepts a string input. For example, "I am as unique as a snowflake" is a valid input.
+** This RPC should return the task ID associated with this unique combination of account ID and provided ID.
+** This RPC does not provide the task ID for only tasks that are scheduled. It can provided the potential task ID for non-scheduled tasks, so long as the non-scheduled tasks are scheduled with the provided account_id and provided_id. 
```

Comment:
These all got formatted on one line
![image](https://user-images.githubusercontent.com/16072671/180031624-aec16d68-ecc7-4d44-92b8-7f0d93d4581c.png)

Also, I think it's just `*` for bulleted list.